modules/integration_aws-rds-common/detectors-rds-common.tf

```diff
@@ -35,8 +35,8 @@ resource "signalfx_detector" "cpu_90_15min" {
 
   program_text = <<-EOF
     signal = data('CPUUtilization', filter=filter('namespace', 'AWS/RDS') and filter('stat', 'mean') and filter('DBInstanceIdentifier', '*') and ${module.filtering.signalflow})${var.cpu_90_15min_aggregation_function}${var.cpu_90_15min_transformation_function}.publish('signal')
-    detect(when(signal > ${var.cpu_90_15min_threshold_critical})).publish('CRIT')
-    detect(when(signal > ${var.cpu_90_15min_threshold_major}) and (not when(signal > ${var.cpu_90_15min_threshold_critical}))).publish('MAJOR')
+    detect(when(signal > ${var.cpu_90_15min_threshold_critical}, lasting=%{if var.cpu_90_15min_lasting_duration_critical == null}None%{else}'${var.cpu_90_15min_lasting_duration_critical}'%{endif}, at_least=${var.cpu_90_15min_at_least_percentage_critical})).publish('CRIT')
```

Comment:
Is that Jinja2 templating? I do not believe it is supported here.

Reply:
Yeah, like here jinja is already used: https://github.com/claranet/terraform-signalfx-detectors/blob/master/modules/integration_aws-rds-common/detectors-gen.tf#L11
already tested and is ok.

Reply:
Oh, this is not Jinja2 templating but [Terraform templating](https://developer.hashicorp.com/terraform/language/expressions/strings#directives). Never used that all that time :facepalm: